shell/platform/windows/settings_plugin_win32.h

```diff
@@ -39,11 +39,15 @@ class SettingsPluginWin32 : public SettingsPlugin {
 
  private:
   void WatchPreferredBrightnessChanged();
+  void WatchTextScaleFactorChanged();
 
   HKEY preferred_brightness_reg_hkey_ = NULL;
+  HKEY text_scale_factor_reg_hkey_ = NULL;
```

Comment:
nit: Use `nullptr` for initializing both HKEYs.

Reply:
done